doc/api/util.markdown

```diff
@@ -335,6 +335,22 @@ util.isError({ name: 'Error', message: 'an error occurred' })
   // false
 ```
 
+Note that this method relies on `Object.prototype.toString()` behavior. It is
+possible to obtain an incorrect result when the `object` argument manipulates
+`@@toStringTag`.
+
+```js
+// This example requires the `--harmony-tostring` flag
```

Comment:
Okay, in that case, this particular thing is applicable to few other functions in `util`. Can we put this as a separate section and link it in all the relevant function descriptions?